api/src/main/java/uk/gov/dhsc/htbhf/dwp/factory/EligibilityResponseFactory.java

```diff
@@ -1,43 +1,71 @@
 package uk.gov.dhsc.htbhf.dwp.factory;
 
+import org.springframework.stereotype.Component;
 import uk.gov.dhsc.htbhf.dwp.entity.Child;
 import uk.gov.dhsc.htbhf.dwp.entity.legacy.LegacyHousehold;
 import uk.gov.dhsc.htbhf.dwp.entity.uc.UCHousehold;
+import uk.gov.dhsc.htbhf.dwp.model.ChildDTO;
 import uk.gov.dhsc.htbhf.dwp.model.EligibilityResponse;
 import uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus;
 
 import java.time.LocalDate;
+import java.util.List;
 import java.util.Set;
+import java.util.stream.Collectors;
 
+/**
+ * Generates an {@link EligibilityResponse} for a given household ({@link LegacyHousehold} or {@link UCHousehold}) and {@link EligibilityStatus}.
+ */
+@Component
```

Comment:
Why does this need to be a component, it has no other dependencies and its methods could all be static?

Reply:
It contains quite a bit of logic which will affect other tests. I found that the eligibility service test was breaking even when all it's mocks were set up correctly.